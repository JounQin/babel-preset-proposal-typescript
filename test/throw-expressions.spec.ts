import { babelExecute, tsExecute } from './helpers'

const proposal = 'throw-expressions'

test(proposal, () => {
  expect(() => tsExecute(proposal)).toThrowErrorMatchingSnapshot()
  expect(() => babelExecute(proposal)).toThrow('Argument required')
  expect(babelExecute(proposal, proposal)).toBe(proposal)
})
