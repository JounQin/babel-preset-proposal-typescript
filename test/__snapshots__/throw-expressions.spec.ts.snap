// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throw-expressions 1`] = `"Unexpected token )"`;
